lib/Events/BeforeMessagesDeletedEvent.php

```diff
@@ -54,7 +54,7 @@ public function getFolderId(): string {
 		return $this->folderId;
 	}
 
-	public function getMessageId(): int {
-		return $this->messageId;
+	public function getMessageId(): array {
```

Comment:
```suggestion
	public function getMessageIds(): array {
```

and please add a docblock for ``@returns int[]``

Reply:
done